build-system/tasks/css/test/bento-css.test.js

```diff
@@ -94,8 +94,8 @@ test(
 );
 
 test(
-  'renames amp-* selectors to bento-* (preserves minification)',
+  'renames amp-* selectors to bento-* (preserves minification in content)',
   renameSelectorsToBentoTagNamesExpected,
   `amp-foo>div:not(.amp-classname){background:red}amp-whatever{color:blue}`,
-  `bento-foo>div:not(.amp-classname){background:red}bento-whatever{color:blue}`
+  `bento-foo > div:not(.amp-classname){background:red}bento-whatever{color:blue}`
```

Comment:
This should be mostly okay since the rest of the CSS remains minified, and the only cases where selectors may have strippable spaces are `,` and `>`.